samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -214,7 +227,11 @@ static void audio_timer_timeout(struct k_work *work)
 		}
 	}
 
-	k_work_schedule(&audio_send_work, K_MSEC(1000U));
+#if defined(CONFIG_LIBLC3)
+	k_work_schedule(&audio_send_work, K_USEC(MAX_FRAME_DURATION_US));
+#else
+	k_work_schedule(&audio_send_work, K_USEC(AUDIO_DATA_TIMEOUT_US));
+#endif
```

Comment:
Ditto with using `IS_ENABLED`. 

If we want to do this correctly, we should use the `bt_codec_qos.interval` from the `lc3_qos` callback as the interval here. 

We already store the `max_sdu` size, so should be easy to store the interval as well

Reply:
I am not familiar with the implementation of codec, if you provide suggested code I can include it in this PR. Or you are welcome to send a follow up PR.

No change done, as `MAX_FRAME_DURATION_US` is currently conditionally compiled hence not able to use `IS_ENABLED`